.github/workflows/ci.yaml

```
@@ -58,13 +58,22 @@ jobs:
       - uses: oNaiPs/secrets-to-env-action@v1
         with:
           secrets: ${{ toJSON(secrets) }}
-      - run: ./rebuild.sh
+      - name: build deps
+        run: |
+          deps/kiam/build.sh
```

```suggestion
          deps/kiama/build.sh
```
I believe the folder is deps/kiama